TerminalDocs/customize-settings/profile-appearance.md

```diff
@@ -147,7 +147,21 @@ This sets the percentage height of the cursor starting from the bottom. This wil
 
 ___
 
-## Background image
+## Background images and icons
```

Comment:
```suggestion
## Background images
```
Terminal won't accept an icon as a background image, only .jpg, .png, and .gif files.